tests/bluetooth/tester/src/btp_ccp.c

```diff
@@ -202,14 +223,88 @@ static void tbs_client_termination_reason_cb(struct bt_conn *conn,
 		err, call_index, reason);
 }
 
+static void tbs_client_read_string_cb(struct bt_conn *conn, int err, uint8_t inst_index,
+				      const char *value)
+{
+	LOG_DBG("TBS Client read string characteristic value cb");
+
+	uint8_t data_len = strlen(value);
+
+	tbs_client_chrc_str_ev(conn, err ? BTP_STATUS_FAILED : BTP_STATUS_SUCCESS, inst_index,
+			       data_len, value);
+}
+
+static void tbs_client_read_val_cb(struct bt_conn *conn, int err, uint8_t inst_index,
+				   uint32_t value)
+{
+	LOG_DBG("TBS Client read characteristic value cb");
+
+	tbs_client_chrc_val_ev(conn, err ? BTP_STATUS_FAILED : BTP_STATUS_SUCCESS, inst_index,
+			       value);
+
+	if (send_ev == 1) {
+		inst_ccid = value;
+
+		tbs_inst = bt_tbs_client_get_by_ccid(conn, inst_ccid);
+
+		tbs_handles.provider_name = tbs_inst->name_sub_params.value_handle;
+		tbs_handles.bearer_uci = tbs_inst->bearer_uci_handle;
+		tbs_handles.bearer_technology = tbs_inst->technology_sub_params.value_handle;
+		tbs_handles.uri_list = tbs_inst->uri_list_handle;
+		tbs_handles.signal_strength = tbs_inst->signal_strength_sub_params.value_handle;
+		tbs_handles.signal_interval = tbs_inst->signal_interval_handle;
+		tbs_handles.current_calls = tbs_inst->current_calls_sub_params.value_handle;
+		tbs_handles.ccid = tbs_inst->ccid_handle;
+		tbs_handles.status_flags = tbs_inst->status_flags_sub_params.value_handle;
+		tbs_handles.bearer_uri = tbs_inst->in_target_uri_sub_params.value_handle;
+		tbs_handles.call_state = tbs_inst->call_state_sub_params.value_handle;
+		tbs_handles.control_point = tbs_inst->call_cp_sub_params.value_handle;
+		tbs_handles.optional_opcodes = tbs_inst->optional_opcodes_handle;
+		tbs_handles.termination_reason = tbs_inst->termination_reason_handle;
+		tbs_handles.incoming_call = tbs_inst->incoming_call_sub_params.value_handle;
+		tbs_handles.friendly_name = tbs_inst->friendly_name_sub_params.value_handle;
+
+		tbs_chrc_handles_ev(&tbs_handles);
+		send_ev = 0;
+	}
+}
+
+static void tbs_client_current_calls_cb(struct bt_conn *conn, int err, uint8_t inst_index,
+					uint8_t call_count, const struct bt_tbs_client_call *calls)
+{
+	LOG_DBG("TBS Client Current Calls cb");
+}
+
+static void tbs_client_cp_cb(struct bt_conn *conn, int err, uint8_t inst_index, uint8_t call_index)
+{
+	LOG_DBG("TBS Client Control Point cb");
```

Comment:
Given that LOG_DBG by default prefixes the log statement with the function name, this could be
```suggestion
	LOG_DBG("");
}

static void tbs_client_cp_cb(struct bt_conn *conn, int err, uint8_t inst_index, uint8_t call_index)
{
	LOG_DBG("");
```